phoenix-ios/phoenix-ios/views/configuration/privacy and security/recovery phrase/RecoveryPhraseView.swift

```diff
@@ -854,6 +874,42 @@ fileprivate struct RecoveryPhraseReveal: View {
 		} // </HStack>
 	}
 	
+	@ViewBuilder
+	func copyButton() -> some View {
+		
+		let xprv = BusinessManager.isTestnet ? "vprv" : "zprv"
+		let xpub = BusinessManager.isTestnet ? "vpub" : "zpub"
+		
+		HStack(alignment: VerticalAlignment.center, spacing: 0) {
+			Spacer()
+				
+			Button {
+				showCopyOptions = true
+			} label: {
+				Text("Copy…").font(.title3)
+			}
+			.confirmationDialog("What would you like to copy?",
+				isPresented: $showCopyOptions,
+				titleVisibility: .automatic
+			) {
+				// Note: confirmationDialog strips all formatting from Text items.
+				// So we don't get to play with fonts or colors here.
+				Button("Recovery phrase (12 words)") {
+					copyRecoveryPhrase()
+				}
+				Button("Account extended private key (\(xprv))") {
```

Comment:
In fact, the xprv export button should be removed. Instead of having 1 secret (the 12-words recovery phrase), Phoenix would now have 2 secrets. It's fine for technical people, but confusing/dangerous for regular users.

This button is only useful to recover funds after a force-close, since we rely on Electrum for that. Which is not good anyway. The correct solution is being able to empty the final wallet directly from Phoenix.

So let's remove this xprv button and merge this PR. We'll then make another PR that support spending from the final wallet. Some work will be needed in lightning-kmp, but that can be done in the next weeks.

Reply:
Done in e43191e